main.py

```diff
@@ -96,3 +98,56 @@ def plugin_unloaded():
         if ref_info:
             ref_view.settings().set('refinfo', ref_info.as_value())
     cli.service.exit()
+
+
+_UPDATE_NPM_MESSAGE = "Warning from TypeScript Plugin: We detect a global \
+npm typescript module and it is outdated. Consider upgrading this module by \
+running \"npm install -g typescipt\". TypeScript Plugin still works \
+regardless."
```

Comment:
Per below comment about version, we should just say something like: "Detected command-line TypeScript compiler version '{0}'. The TypeScript Sublime Text plugin is using compiler version '{1}'. There may be differences in behavior between releases. 

To update your command-line TypeScript compiler to the latest release, run 'npm update -g typescript'."